source/extensions/filters/http/grpc_json_transcoder/BUILD

```diff
@@ -45,7 +45,7 @@ envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
-    security_posture = "unknown",
+    security_posture = "robust_to_untrusted_downstream_and_upstream",
```

Comment:
definitely not robust to upstream, there's couple TODO there.

Reply:
Hi Lizan, 

Which TODO are you referring to?  In the grpc_transcoding repo,  there are 4 TODO, 
3 of them are path_matcher, one is request_weaver.   It seems that all of them are downstream related. 

Upstream should be simpler than downstream, it just translate response from proto to JSON,  and mainly calling ::google::protobuf::util::BinaryToJsonStream function.  We can safely assume this function is secure since it is protobuf code, it is a production code

I would say we it is already "robust_to_untrusted_upstream".  

To make it "robust_to_untrusted_downstream" is a hard task

Reply:
It is opposite, we don't handle BinaryToJsonStream error statuses, and we should test it with malformed gRPC framing upstream etc.

Reply:
The filter does infinite buffering internally.  In order to be robust to untrusted peers it needs a configurable buffer limit.  Ideally this config would be required or have a reasonable default (not infinite)

Reply:
@antoniovicente can you file an issue on this so we can track?